samples/tfm_integration/psa_crypto/README.rst

```diff
@@ -197,6 +197,12 @@ We need to reset the board manually after flashing the image to run this code.
 On nRF5340 and nRF9160:
 =======================
 
+.. note::
+   To be able to create a initial attestation token (IAT) the device has to be provisioned,
+   by generating a Hardware Unique key (HUK) and an identity key.
+   An example how to do this, can be found in the TF-M provisioning sample of NCS.
```

Comment:
Create a reference?

Reply:
Not possible as this is in sdk-zephyr :(

Reply:
We use intersphinx to allow links between docsets. E.g. from sdk-nrf, you can say

```
:ref:`zephyr:some_zephyr_ref`
```

I believe it may be possible from sdk-zephyr to say:

```
:ref:`nrf:some_nrf_ref`
```

Reply:
Then the doc build in the CI here fails, but can try it again

Reply:
Never mind, then.